optimar/.htaccess

```diff
@@ -0,0 +1,36 @@
+# Name of the project: OPTiMAR
+# Description: Ontology modeling structural elements, sensors, events, and failures in offshore wind platforms, including measurement data and involved actors.
+# Contacts:
+# - Alejandro Flores <afloresb@usal.es> - GitHub: https://github.com/alexespz
+# - Alberto Montero <albertomonfdez@usal.es> - GitHub: https://github.com/tobeal
+# - Cristina Aceta <cristina.aceta@tekniker.es> - GitHub: https://github.com/cristinacm
+# - Oihana Garcia <oihana.garcia@tekniker.es>
+
+
+## Turn off MultiViews
+Options -MultiViews
+
+# Rewrite engine setup
+RewriteEngine On
+
+# Rewrite rule to serve HTML content
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^$ https://bisite.github.io/OPTIMAR-Ontology/docs/index.html [R=303,L]
+
+# Rewrite rule to serve RDF/XML content if requested
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} application/owl\+xml
+RewriteRule ^$
+https://bisite.github.io/OPTIMAR-Ontology/ontology/OPTiMAR-v1.0.ttl [R=303,L]

```

Comment:
add these in the same line: 
`RewriteRule ^$ https://bisite.github.io/OPTIMAR-Ontology/ontology/OPTiMAR-v1.0.ttl [R=303,L]`